### PR TITLE
add notes from experimentation

### PR DESCRIPTION
**Pull request by MikeBazMSFT**

### CODE REVIEW

**Comment by jborsecnik** [comment]
@ronmiab, 

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by ronmiab** [comment]
@jborsecnik - The proposed changes are under review.

**Review by ronmiab — APPROVED** [review]
I have reached out to the PM to verify the changes and they have approved.

**Comment by ronmiab** [comment]
Please #sign-off

File: azure-stack/hci/deploy/single-server.md [hunk 1]
```diff
@@ -46,6 +46,8 @@ Here are the steps to install the Azure Stack HCI OS on a single server, create
    ```powershell
    Enable-ClusterStorageSpacesDirect -CacheState Disabled 
    ```
+   
+   Note that the `New-Cluster` command will also require the `StaticAddress` parameter if the node is not using DHCP for its IP address assignment.  This parameter should be supplied with a new, available IP address on the node's subnet.
```
Comment (Jak-MS) [inline azure-stack/hci/deploy/single-server.md:50]:
```suggestion
   > [Note]
   > The `New-Cluster` command will also require the `StaticAddress` parameter if the node is not using DHCP for its IP address assignment.  This parameter should be supplied with a new, available IP address on the node's subnet.
```

**Pull request merged**